docs/open-id-connect.rst

```diff
@@ -5,50 +5,54 @@ OpenID Connect (Beta)
 Overview
 -------------
 
-This endpoint provides the ability to authenticate users on onadata using an OpenID Connect provider, as such users can be created or logged in to the platform using OpenID Connect Providers such as Microsoft.
+This includes the ability to authenticate users on onadata using an OpenID Connect provider, as such users can be created or logged in to the platform using OpenID Connect Providers such as Microsoft.
 
-The OpenID Connect endpoint has only been tested with the Microsoft Platform.
+The |OpenIDConnectlibrary| has currently only been tested with the Microsoft Platform.
+
+.. |OpenIDConnectlibrary| raw:: html
+
+    <a href="https://github.com/onaio/ona-oidc"
+    target="_blank">OpenID Connect library</a>
 
 When utilizing the OpenID Connect flow ensure the ``given_name``, ``family_name`` and ``email`` claims are available in the ID Token provided by the OpenID Connect provider.
 
 Enabling OpenID Connect on onadata
------------------------------------------------
+----------------------------------
 
-In order to enable OpenID Connect authentication for a particular provider on the platform set the ``OPENID_CONNECT_PROVIDERS`` variable within your onadata ``local_settings.py``. The ``OPENID_CONNECT_PROVIDERS`` variable should be a ``dict``.
+In order to enable OpenID Connect authentication for a particular provider on the platform set `OPENID_CONNECT_VIEWSET_CONFIG` and `OPENID_CONNECT_AUTH_SERVERS` variables within your onadata ``local_settings.py``.
```

Comment:
Kindly include a link to the ona-oidc package just in case

Reply:
That was already added [here](https://github.com/onaio/onadata/pull/2096/files/1e4adf07364bc3f4f745345c4554411478d5003c#diff-fcc8f6c2da89aaaf7108cbacaef82de4a3b4112ad9b7e3a661c6d179818fe143R10)

Reply:
Ohh cool, missed that. We should also make sure the documentation is in the index https://github.com/onaio/onadata/blob/master/docs/index.rst